src/appimagetool.c

```diff
@@ -1,6 +1,6 @@
 /**************************************************************************
  * 
- * Copyright (c) 2004-19 Simon Peter
+ * Copyright (c) 2004-21 Simon Peter
```

Comment:
You're not the only one who's authored code on this tool...

Reply:
Please feel free to amend. I merely updated the timespan.